test/members_page_test.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+require_relative './test_helper'
+require_relative '../lib/members_page'
+
+describe MembersPage do
+  around { |test| VCR.use_cassette(File.basename(url), &test) }
+  let(:response)  { MembersPage.new(response: Scraped::Request.new(url: url).response) }
+  
+  describe 'first page' do
+    let(:url) { 'http://www.congreso.es/portal/page/portal/Congreso/Congreso/Diputados/DiputadosTodasLegislaturas' }
+    it 'should contain the expected number of member urls' do
+      response.member_urls.count.must_equal 25
+    end
+
+    it 'should contain the expected url' do
+      response.member_urls.first.must_equal 'http://www.congreso.es/portal/page/portal/Congreso/Congreso/Diputados/BusqForm?_piref73_1333155_73_1333154_1333154.next_page=/wc/fichaDiputado&idDiputado=87&idLegislatura=10'
```

Comment:
I'm a little surprised to see the "_piref" here. I thought the plan was to remove this from URLs where possible? Has that plan changed, or is that something that will happen later?